src/main/java/ObjectifyInit.java

```diff
@@ -16,7 +16,8 @@ public void contextInitialized(ServletContextEvent event) {
                 .build()
                 .getService()));
     // To deploy, everything above this must be commented out
-    // and replaced with only ObjectifyService.init();
+    // and line directly below show be uncommented
+    // ObjectifyService.init();
```

Comment:
I think this is not a good situation to be in where you have to comment out lines of code to make it work locally versus when deployed. We should talk about a cleaner way to do this.